Design_Patterns/Builder/pizza_builder.py

```diff
@@ -25,21 +29,33 @@ def add_filling(self):
         self.pizza.filling = ['Salami', 'Tomatoes', 'Cheese']
         return self
 
+    def build(self):
+        return self.pizza
+
 
 class Pizza:
-    def __init__(self):
+    def __init__(self, strategy):
```

Comment:
Pizza should not posses any logic related to the strategy it's built, nor the Builder.
The Director is the proper entity to resent a strategy (i.e. different directors may have different strategies for making a pizza)